pkg/frontend/frontend.go

```diff
@@ -179,6 +179,12 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 	s.Methods(http.MethodPost).HandlerFunc(f.postOpenShiftClusterCredentials).Name("postOpenShiftClusterCredentials")
 
 	// Admin actions
+	s = r.
+		Path("/admin/version").
```

Comment:
If we do this I am inclined to put it unauthenticated under /version like k8s.
But I'm not sure to what extent we actually need this PR when we have better rollout processes?

Reply:
So I am happy to ditch this, just at the point of running GAs it seemed like a good idea to see
1. the RP version
2. what version of openshift it would upgrade to (i'll add that and see what you think)

I'd be interested to hear what we could do instead..

Reply:
I think this is more useful now:
```
curl  -k "https://localhost:8443/version"
{"provider":"d5051bba-dirty","openshift":"4.3.10"}
```